app/components/SignUpForm.tsx

```diff
@@ -0,0 +1,115 @@
+"use client";
+import { useState } from "react";
+import Input from "./Input";
+import Button from "./Button";
+import Letter from "./Icons/Letter";
+import Lock from "./Icons/Lock";
+import User from "./Icons/User";
+import LoadingSpinner from "./Icons/LoadingSpinner";
+import Confetti from "./Icons/Confetti";
```

Comment:
see previous comment of the `Icons/index.ts`
so that you can use `import { User, Letter, Lock, LoadingSpinner, Confetti } from "./Icons";` and improve readability